test/e2e/specs/fse/fse__temp-smoke-test.ts

```diff
@@ -24,12 +24,46 @@ declare const browser: Browser;
  * The goal here is to catch major breaks with the integration -- i.e. Calypso navigation no long working,
  * or getting a WSOD when trying to load the editor.
  */
+
 describe( DataHelper.createSuiteTitle( 'Site Editor Smoke Test' ), function () {
 	let page: Page;
 	let fullSiteEditorPage: FullSiteEditorPage;
 
 	const features = envToFeatureKey( envVariables );
-	const accountName = getTestAccountByFeature( { ...features, variant: 'siteEditor' } );
+	const accountName = getTestAccountByFeature( { ...features, variant: 'siteEditor' }, [
+		// None of our CoBlocks users use block themes, so we need to fall back to the default Gutenberg users
+		// if COBLOCKS_EDGE is set.
+		{
+			gutenberg: 'stable',
+			coblocks: 'edge',
+			siteType: 'simple',
+			variant: 'siteEditor',
+			accountName: 'siteEditorSimpleSiteUser',
+		},
+		{
+			gutenberg: 'edge',
+			coblocks: 'edge',
+			siteType: 'simple',
+			variant: 'siteEditor',
+			accountName: 'siteEditorSimpleSiteEdgeUser',
+		},
+		{
+			gutenberg: 'stable',
+			coblocks: 'edge',
+			siteType: 'atomic',
+			variant: 'siteEditor',
+			accountName: 'siteEditorAtomicSiteUser',
+		},
+		{
+			gutenberg: 'edge',
+			coblocks: 'edge',
+			siteType: 'atomic',
+			variant: 'siteEditor',
+			accountName: 'siteEditorAtomicSiteEdgeUser',
+		},
+	] );
+
+	console.log( accountName );
```

Comment:
**chore:** We can remove this. If we want to continue logging this, we can add a message.

Reply:
Good eye, didn't see that.

Also @dpasque, wouldn't this be easier if we swap the CoBlocks user out to a block-based theme?